coalib/bears/requirements/PythonRequirement.py

```diff
@@ -5,6 +5,37 @@ class PythonRequirement(PackageRequirement):
     """
     This class is a subclass of ``PackageRequirement``, and helps specifying
     requirements from ``pip``, without using the manager name.
+
```

Comment:
It's confusing because it's not in the right place where it "should" be.
It's like reading Geography in a History book :P